vignettes/ebola.Rmd

```diff
@@ -0,0 +1,615 @@
+link-citations: true
+vignette: >
+  %\VignetteIndexEntry{Getting in depth with epidist}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+bibliography: references.bib
+---
+
+```{r setup, include=FALSE}
+# exclude compile warnings from cmdstanr
+knitr::opts_chunk$set(
+  fig.path = "figures/epidist-",
+  cache = TRUE,
+  collapse = TRUE,
+  comment = "#>",
+  message = FALSE,
+  warning = FALSE,
+  error = FALSE
+)
+```
+
+The 2014-2016 outbreak of Ebola in West Africa began in Guinea, and quickly spread to neighbouring Liberia and Sierra Leone.
+It was the largest outbreak in the disease's history, and resulted in over 28,600 infections and 11,325 deaths [@who_ebola_2014_2016].
+
+In this vignette, we use the `epidist` package to analyze line list data from the outbreak in Sierra Leone.
+This data was collated by @fang2016transmission.
+We provide the data in the `epidist` package via `?sierra_leone_ebola_data`.
+
+In analyzing this data, we demonstrate the following features of `epidist`:
+
+1. Fitting district-sex stratified delay distribution estimates.
+2. Fitting models with lognormal and gamma delay distributions.
+3. Post-processing and plotting functionality using the integration of `brms` functionality with the [`tidybayes`](http://mjskay.github.io/tidybayes/) package.
+4. Selecting between fitted models using the [`loo`](https://mc-stan.org/loo/) package.
+
+The packages used in this article are:
+
+```{r load-requirements}
+set.seed(1)
+
+library(epidist)
+library(data.table)
+library(purrr)
+library(ggplot2)
+library(sf)
+library(tidybayes)
+library(modelr)
+```
+
+Note that, for users new to `epidist`, before reading this article we recommend beginning with the vignette "[Getting started with `epidist`](http://epidist.epinowcast.org/articles/epidist.html)".
+
+# Data preparation
+
+We begin by loading the Ebola line list data:
+
+```{r}
+data("sierra_leone_ebola_data")
+```
+
+The data has `r nrow(sierra_leone_ebola_data)` rows, each corresponding to a unique case report `id`.
+The columns of the data are the individuals name (retracted, and hence can be removed), age, sex, the dates of Ebola symptom onset and positive sample, and their district and chiefdom.
+
+```{r}
+sierra_leone_ebola_data <- sierra_leone_ebola_data |>
+  dplyr::select(-name) |>
+  dplyr::mutate(case = as.integer(id), .keep = "unused")
+
+head(sierra_leone_ebola_data)
+
+fraction <- 5
+ndistrict <- length(unique(sierra_leone_ebola_data$district))
+```
+
+Figure \@ref(fig:ebola-outbreak) shows the progression of the epidemic in each district.
+(In this figure, we filter down to every `r fraction`th case in order to avoid overplotting.)
+We can see that the start time and course of the epidemic varies between districts.
+
+(ref:ebola-outbreak) Primary and secondary event times for every `r fraction`th case, over the `r ndistrict` districts of Sierra Leone.
```

Comment:
What delay are we estimating (onset to report?)

This should probably be introduced earlier.

Reply:
Have been more specific in f45723cc